extra/mariabackup/backup_mysql.cc

```diff
@@ -124,6 +124,11 @@ xb_mysql_connect()
 	mysql_options(connection, MYSQL_OPT_PROTOCOL, &opt_protocol);
 	mysql_options(connection,MYSQL_SET_CHARSET_NAME, "utf8");
 
+    if(opt_bind_address && *opt_bind_address) {
+        mysql_options(connection, MYSQL_OPT_BIND, opt_bind_address);
+    }
```

Comment:
just watch the whitespace formatting, normally what you did is right, however there's tabs in this section of file so best not to mess that up too much (unless you where rewriting most of a function, in which its perfectly ok to change the entire function)..